frappe/desk/doctype/todo/todo.json

```diff
@@ -1,13 +1,14 @@
 {
  "actions": [],
- "autoname": "hash",
+ "autoname": "field:subject",
```

Comment:
this makes subject unique, which kind of too restrictive. better to keep hash only and make the subject as subject_field and title_field